core/src/com/goonsquad/galactictd/systems/initialization/ScoreScreenInitSystem.java

```diff
@@ -117,4 +124,29 @@ private int createGreenShip() {
         speed.unitsPerSecond = shipSpeedPerSecond;
         return greenShip;
     }
+
+    private void createText() {
+        int label = archetypeBuilder.buildArchetype(ScoreScreenArchetypeBuilder.textLabel);
+
+        Text labelText = textComponentMapper.get(label);
+        labelText.text = "High Scores:";
+        labelText.font = gameInstance.assets.manager.get("nixie48.ttf", BitmapFont.class);
+
+        Position labelPosition = positionComponentMapper.get(label);
+        labelPosition.x = textPosition.x;
+        labelPosition.y = textPosition.y * 15;
+
+        int textHeightFactor = 15;
+        for (int i = 0; i < 5; i++) {
+            label = archetypeBuilder.buildArchetype(ScoreScreenArchetypeBuilder.textLabel);
+            labelText = textComponentMapper.get(label);
+            labelText.text = "0";
+            labelText.font = gameInstance.assets.manager.get("nixie48.ttf", BitmapFont.class);
+
+            labelPosition = positionComponentMapper.get(label);
+            labelPosition.x = textPosition.x * 3;
+            labelPosition.y = textPosition.y * textHeightFactor;
+            textHeightFactor -= 2;
+        }
```

Comment:
Some explanatory comments about these magic numbers would be great (consider using constants or making them configurable as well).